### PR TITLE
Add documentation for aggregator env vars

### PR DESCRIPTION
**Pull request by jacque006**
## What is this PR doing?
Adding documentation for aggregator env vars

## How can these changes be manually tested?
:eyes:

## Does this PR resolve or contribute to any issues?
No

## Checklist
- [x] I have manually tested these changes
- [x] Post a link to the PR in the group chat

## Guidelines
- If your PR is not ready, mark it as a draft
- The `resolve conversation` button is for reviewers, not authors
  - (But add a 'done' comment or similar)

### CODE REVIEW

File: aggregator/README.md [hunk 1]
```diff
@@ -29,6 +29,34 @@ you might have:
...
+| USE_TEST_NET                 | false                                                              | Whether to set all transaction's `gasPrice` to 0. Workaround for some networks                                                                                       |
+| ORIGIN                       | http://localhost:3000                                              | The origin for the aggregator client. Used only in manual tests                                                                                                      |
+| PORT                         | 3000                                                               | The port to bind the aggregator to                                                                                                                                   |
+| NETWORK_CONFIG_PATH          | ../contracts/networks/local.json                                   | Path to the netowrk config file, which contains information on deployed BLS Wallet contracts                                                                         |
```
Comment (blakecduncan) [inline aggregator/README.md]: nit: typo in `network`

File: aggregator/README.md [hunk 2]
```diff
@@ -29,6 +29,34 @@ you might have:
...
+| MAX_AGGREGATION_SIZE         | 12                                                                 | Maximum number of actions from bundles which will be aggregated together for submission on chain                                                                     |
+| MAX_AGGREGATION_DELAY_MILLIS | 5000                                                               | Maximum amount of time in milliseconds aggregator will wait before submitting bundles on chain                                                                       |
+| MAX_UNCONFIRMED_AGGREGATIONS | 3                                                                  | Maximum unconfirmed bundle aggregations that will be submitted on chain. Multiplied with `MAX_AGGREGATION_SIZE` to determine maximum of unconfirmed on chain actions |
+| LOG_QUERIES                  | false                                                              | Whether to print Postgres quries in event log.`TEST_LOGGING` must be enabled                                                                                         |
```
Comment (blakecduncan) [inline aggregator/README.md]: nit: typo in `queries`

**Review by blakecduncan — APPROVED** [review]
Probably good for someone with more aggregator experience to review but this all sounds good to me.  Thanks for adding!

File: aggregator/README.md (hunk 1 above)
Comment (jacque006) [reply aggregator/README.md]: Fixed

File: aggregator/README.md (hunk 2 above)
Comment (jacque006) [reply aggregator/README.md]: Fixed

**Comment by jacque006** [comment]
@voltrevo can take a look when he's back online, happy to make changes. Will merge and update then if needed.

**Pull request merged**